src/java.xml/share/classes/module-info.java

```diff
@@ -416,6 +423,30 @@
  * </li>
  * </ul>
  *
+ * <h2 id="IN_CF">JAXP Configuration Files</h2>
+ * The JDK provides two <a href="#Conf_CF">JAXP Configuration Files</a>:
+ *
+ * <ul>
+ * <li id="IN_CF_DEFAULT"><a href="#Conf_CF_Default">{@code jaxp.properties}</a>:
+ * the default configuration that the JDK uses to set property values when XML
+ * factories are initiated.</li>
+ *
+ * <li id="IN_CF_STRICT">{@code jaxp-strict.properties}: a configuration that
+ * contains more restrictive settings than the default {@code jaxp.properties}.
+ * In particular:
+ *     <ul>
+ *     <li><a href="#JDKCATALOG_RESOLVE">JDKCatalog Resolve</a> is on "strict" setting </li>
+ *     <li><a href="#ExtFunc">Extension Functions</a> are disabled </li>
+ *     <li><a href="#EELimit">JAXP Limits</a> are set to smaller numbers </li>
+ *     </ul>
+ *
+ * This file allows deployments to test the more secure/strict behavior,
```

Comment:
I think it might be better to reduce this paragraph down to just say something like "Deploying with this configuation prevents processors from unknowingly making outbound network connections to fetch DTDs, or process XML that makes use of extension functions."

We could say that a future JDK release may use a strict configuration by default but that opens the door to questions as to whether the system property is needed, whether jaxp.propeteries is going away, so maybe better to leave that out for now.